Documentation/contributing.rst

```diff
@@ -181,6 +181,23 @@ If for some reason, running of the provisioning script fails, you should bring t
 
     $ vagrant halt
 
+Packer-CI-Build
+^^^^^^^^^^^^^^^^
+
+In cilium, we use a custom base box with a bunch of pre-installed libraries and
+tools that we need to our daily workflow. That base box is built with `Packer
+<https://www.packer.io/>`_ and it's hosted in the `packer-ci-build
+<https://github.com/cilium/packer-ci-build>`_ project.
+
+This base box can be updated and publish from `Jenkins
+<https://jenkins.cilium.io/job/cilium/job/packer-ci-build/job/master/>`_, where
+a new rebuild will update the `Vagrantcloud <https://app.vagrantup.com/cilium>`_
```

Comment:
... New builds of the image will be pushed to `Vagrant Cloud...`. The version of the image corresponds to the `BUILD_ID...` on Jenkins.